docs/source/hello_world.rst

```diff
@@ -0,0 +1,101 @@
+Basics of Study Construction
+=============================
+
+Now that you're acquainted with Maestro's interface running a pre-made example, we'll walk you through the basics of making a simple "Hello, World" specification of your own. This page will walk you through the following:
+
+- A single step "Hello World" without parameterization.
+- An introduction to a single parameter "Hello World" study.
+- An introduction to a multi-parameter "Hello World" study.
+- Adding a "farewell" step to "Hello World".
+
+Maestro's default study description uses general YAML notation, which stands for "Yet Another Markup Language" and is a standard data serialization language. For more information on the YAML language, head `here <https://yaml.org/spec/1.2/spec.html>`_ to learn more.
+
+Creating a Single Step Study
+*****************************
+
+To start, we will walk through constructing a single step "Hello World" study that simply echoes "Hello, World!" to a file. The first step is to name your study -- in this case we'll settle for something simple and just call our study "Hello World". In your editor of choice, begin by adding the following:
+
+.. code-block:: yaml
+    :linenos:
+
+    description:
+        name: hello_world
+        description: A simple 'Hello World' study.
+
+
+.. note:: The `description` block is a required section in every study and has two required keys: name, and description. You may add other keys to the description section, but Maestro will not check for them.
+
+The next section to add will be the `study` section which will only contain a single step. Below the `description` section in the study file you've created add the following block:
+
+.. code-block:: yaml
+    :linenos:
+
+    study:
+        - name: hello_world
+          description: Build the serial version of LULESH.
+          run:
+              cmd: |
+                echo "Hello, World!" > hello_world.txt
+
+
+.. note:: The `-` denotes a list item in YAML, which means if you wanted to add more steps you could. For now, though, we will keep it simple with one.
```

Comment:
Double backquotes can be used to render inline highlighting. It might be useful here to reference the `-`.